ReactCommon/hermes/inspector/tools/run_msggen

```diff
@@ -12,7 +15,8 @@ FBSOURCE=$(hg root)
 CLANG_FORMAT="${FBSOURCE}/tools/third-party/clang-format/clang-format"
 SIGNEDSOURCE="${FBSOURCE}/tools/signedsource"
 
-cd "${DIR}/msggen"
+popd >/dev/null || exit
+cd "packages/hermes-inspector-msggen"
```

Comment:
I would suggest we `pushd "${DIR}/msggen"` (or `$ROOT/packages/hermes-inspector-msggen`) and `popd` at the end of this script so the user is back in the folder where the script was invoked.

Reply:
Sure, updated